Readability.js

```diff
@@ -152,10 +152,17 @@ Readability.prototype = {
    *
    * @param  NodeList nodeList The NodeList.
    * @param  Function fn       The iterate function.
+   * @param  Boolean  backward Whether to use backward iteration.
    * @return void
    */
-  _forEachNode: function(nodeList, fn) {
-    return Array.prototype.forEach.call(nodeList, fn, this);
+  _forEachNode: function(nodeList, fn, backward) {
```

Comment:
If every caller of this that uses the `backward` param is doing so because it removes items, maybe it makes more sense to make a separate function for doing this? Something like `_removeNodes(tagName, filterFn)` that always iterates backwards and removes the item if `filterFn` returns true/false? That makes it more obvious when we need to use one function and one the other. Could also rename this function something like `iterateStaticDOM` or whatever.